sdk/quantum/quantum-jobs/README.md

```diff
@@ -1,41 +1,209 @@
+## Azure Quantum Jobs client library for JavaScript
 
 This package contains an isomorphic SDK for QuantumJobClient.
 
+Azure Quantum is a Microsoft Azure service that you can use to run quantum computing programs or solve optimization problems in the cloud. Using the Azure Quantum tools and SDKs, you can create quantum programs and run them against different quantum simulators and machines. You can use the Azure.Quantum.Jobs client library to:
+
+- Create, enumerate, and cancel quantum jobs
+- Enumerate provider status and quotas
+
+  [Source code][source] | [API reference documentation](https://docs.microsoft.com/qsharp/api/) | [Product documentation](https://docs.microsoft.com/azure/quantum/)
+
 ## Getting started
 
+This section includes everything a developer needs to install and create their first client connection _very quickly_.
+
 ### Install the package
 
+Install the Azure Quantum Jobs client library for Javascript with `npm`:
+
 ```bash
 npm install @azure/quantum-jobs
 ```
 
 ### Prerequisites
 
 - Node.js version 8.x.x or higher
+- [Azure subscription](https://azure.microsoft.com/free/)
+- [Azure Quantum Workspace][workspaces]
+
+### Authenticate the client
+
+To authenticate with the service, you can use [DefaultAzureCredential](https://docs.microsoft.com/dotnet/api/azure.identity.defaultazurecredential?view=azure-dotnet). This will try different authentication mechanisms based on the environment (e.g. Environment Variables, ManagedIdentity, CachedTokens) and finally it will fallback to InteractiveBrowserCredential.
+
+The client also allows the user to override the above behavior by passing their own [TokenCredential](https://docs.microsoft.com/dotnet/api/azure.core.tokencredential?view=azure-dotnet).
+
+`TokenCredential` is the default Authentication mechanism used by Azure SDKs.
 
 ## Key concepts
 
-**TO BE FILLED**
+`QuantumJobClient` is the root class to be used to authenticate, and create, enumerate, and cancel jobs.
+
+`JobDetails` contains all the properties of a job.
+
+`ProviderStatus` contains status information for a provider.
+
+`QuantumJobQuota` contains quota properties.
 
 ## Examples
 
-**TO BE FILLED**
+### Create the client
+
+Create an instance of the QuantumJobClient by passing in these parameters:
+
+- [Subscription Id][subscriptions] - looks like XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX and can be found in your list of subscriptions on azure
+- [Resource Group Name][resource-groups] - a container that holds related resources for an Azure solution
+- [Workspace Name][workspaces] - a collection of assets associated with running quantum or optimization applications
+- [Location][location] - choose the best data center by geographical region
+- [Storage Container Name][blob-storage] - your blob storage
+- [Credential][credentials] - used to authenticate
+
+```Javascript Snippet
+    const credential = new DefaultAzureCredential();
+
+    // Create a QuantumJobClient
```

Comment:
const subscriptionId = process.env.SUBSCRIPTION_ID || "";
    const resourceGroupName = process.env.RESOURCE_GROUP_NAME || "";
    const workspaceName = process.env.WORKSPACE_NAME || "";
    const storageContainerName = process.env.STORAGE_CONTAINER_NAME || "";
    const location = process.env.LOCATION || "";; //"your_location";
    const endpoint = "https://" + location + ".quantum.azure.com";

Reply:
I'll update the samples in the next round, following the samples templates with TS and JS, and adding things like this.
Thanks for pointing it out.